lib/gruff/bar.rb

```diff
@@ -76,6 +76,14 @@ def hide_labels?
     @hide_labels
   end
 
+  def hide_left_label_area?
```

Comment:
Changes in bar.rb are the same ones made in PR #450, which should make the PRs compatible for merging.